### ISSUE TITLE
Understanding Blockchain and Decentralized Applications

### ISSUE THREAD

**Issue by satwikkansal** [WTM-NewDelhi/Proposals#4]
**Abstract**

Understanding the concept of blockchain and decentralized applications by actually implementing one with Python.

**About**

- What is blockchain?
- ELI5 explanation of blockchain
- Implementing blockchain with Python (with Proof of Work algorithm)
- Blockchain in action (demonstrating mining, consensus, etc)
- Ethereum, Smart contracts, Proof of Stake and beyond

**Format**

Hands-on session

**Pre-requisites**
- Laptop with Python installed to follow along.
- Basic familiarity with programming.

**Expected duration**

90 mins

**Level**

Beginner

**Previous Experience/Sample ppt**

#TODO: Add links
- PyCon India 2018 (Speaker)
- EuroPython 2017 (Invited as a Speaker)
- OSS DTU (Instructor)

Sample code [here](https://github.com/satwikkansal/ibm_blockchain)

**Speaker Bio**

I'm a freelance Software Developer experience with Data Science and Blockchain technologies. Also a final year Software Engineering undergrad at Delhi Technological University.

**Profile**

[Website](https://satwikkansal.xyz)  | [Linkedin](https://linkedin.com/in/satwikkansal)

**How will it help developers/students**

The session is intended to familiarize the attendees with the fundamentals of blockchain and it's applications. Though there are a lot of good resources covering the theoretical side of the technology, a very few actually delve into practical implementation. By the end of the session, participants would have a decent understanding of blockchain, smart contracts, and the jargons used in the crypto world.

**Comment by anubhakushwaha** [comment]
Closing, talk addressed at IWD-18, [here](https://github.com/WTM-NewDelhi/IWD-18Slides)
